docs/references.md

```diff
@@ -1,20 +1,56 @@
 # References
 
+## Getting started
+
+- Starter pack:
+    - [Install Using Composer](getting_started/install_using_composer.md)
+    - [Bundles starter pack](getting_started/bundles_starter_pac.md)
+    - [First steps](getting_started/first_steps.md)
+    - [eZ Platform beginner tutorial](tutorials/platform_beginner/building_a_bicycle_route_tracker_in_ez_platform.md)
+    - [eZ Enterprise beginner tutorial](tutorials/enterprise_beginner/ez_enterprise_beginner_tutorial_-_its_a_dogs_world.md) 
+    
+## Back end
+
 - API:
     - [REST API reference](https://github.com/ezsystems/ezpublish-kernel/blob/master/doc/specifications/rest/REST-API-V2.rst)
-- Content model and rendering:
+    - [Public PHP API](api/public_php_api.md)
     - [Field Types reference](api/field_type_reference.md)
+- Content model and rendering:
     - [Twig functions reference](guide/twig_functions_reference.md)
     - [Twig helpers list](guide/content_rendering.md#twig-helper)
     - [Available image variation filters](guide/images.md#available-filters)
     - [Available SiteAccess matchers](guide/siteaccess.md#available-matchers)
     - [Available View Matchers](guide/content_rendering.md#available-matchers)
-- Repository:
-    - [Services: Public API reference](guide/repository.md#services-public-api)
-    - [Signals reference](guide/signalslots.md#signals-reference)
 - Permissions:
     - [Limitation reference](guide/limitations.md)
     - [Role Policies map](guide/permissions.md#available-policies)
+    - [Custom Policies](guide/custom_policies.md)
```

Comment:
This is not a reference strictly speaking, but an instruction on how to create a custom policy.